pkg/script/delegated_runner.go

```diff
@@ -94,7 +94,8 @@ func (a *Runner) SendBuild(b *brigade.Build) error {
 }
 
 // SendScript converts a script into a Brigade Build object and submits it
-func (a *Runner) SendScript(projectName string, data []byte, event, commitish, ref string, payload []byte, logLevel string) error {
+// TODO: ripe for refactor to reduce amount of params... create/use Script struct?
```

Comment:
Yeah.. or perhaps go for `SendBuild(projectName string, build *brigade.Build) error {`
because it's almost `brigade.Build` or omitting this function at all?

Reply:
@mumoshu I've decided to create a separate ticket for this work, as I was reminded about `rerun` as well.  My thoughts on the refactor are in https://github.com/brigadecore/brigade/issues/1021